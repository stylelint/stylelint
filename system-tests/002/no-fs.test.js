'use strict';

const stylelint = require('../../lib');
const { caseConfig, caseCode, prepForSnapshot } = require('../systemTestUtils');

const CASE_NUMBER = '002';

it('no-fs - invalid twbs buttons and their config', async () => {
	expect(
		prepForSnapshot(
			await stylelint.lint({
				code: await caseCode(CASE_NUMBER, 'scss'),
				config: await caseConfig(CASE_NUMBER),
				syntax: 'scss',
			}),
		),
	).toMatchSnapshot();
}, 10000);
