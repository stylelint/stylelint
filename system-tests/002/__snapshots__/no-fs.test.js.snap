// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`no-fs - invalid twbs buttons and their config 1`] = `
Object {
  "errored": true,
  "output": Array [
    Object {
      "deprecations": Array [],
      "errored": true,
      "invalidOptionWarnings": Array [],
      "parseErrors": Array [],
      "warnings": Array [
        Object {
          "column": 10,
          "line": 79,
          "rule": "color-named",
          "severity": "error",
          "text": "Unexpected named color \\"white\\" (color-named)",
        },
        Object {
          "column": 1,
          "line": 118,
          "rule": "selector-no-qualifying-type",
          "severity": "error",
          "text": "Unexpected qualifying type selector (selector-no-qualifying-type)",
        },
        Object {
          "column": 3,
          "line": 123,
          "rule": "selector-no-qualifying-type",
          "severity": "error",
          "text": "Unexpected qualifying type selector (selector-no-qualifying-type)",
        },
      ],
    },
  ],
  "results": Array [
    Object {
      "deprecations": Array [],
      "errored": true,
      "ignored": undefined,
      "invalidOptionWarnings": Array [],
      "parseErrors": Array [],
      "warnings": Array [
        Object {
          "column": 10,
          "line": 79,
          "rule": "color-named",
          "severity": "error",
          "text": "Unexpected named color \\"white\\" (color-named)",
        },
        Object {
          "column": 1,
          "line": 118,
          "rule": "selector-no-qualifying-type",
          "severity": "error",
          "text": "Unexpected qualifying type selector (selector-no-qualifying-type)",
        },
        Object {
          "column": 3,
          "line": 123,
          "rule": "selector-no-qualifying-type",
          "severity": "error",
          "text": "Unexpected qualifying type selector (selector-no-qualifying-type)",
        },
      ],
    },
  ],
}
`;
